src/main/java/Duke.java

```diff
@@ -1,10 +1,45 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello, I'm Duke!\n" + "What would you like to do today?");
+        Scanner response = new Scanner(System.in);
+        String line = response.nextLine();
+        int counter = 0;
+        String[] items = new String[101]; // 1-based
+        String[] check = new String[101]; // 1-based
+
+        while (!(line.equals("bye"))) {
+            if (line.equals("list")) {
+                System.out.println("...\n" + "Here are the tasks in your list:");
+                for (int a = 1; a <= counter; a++) {
+                    String checkOrNot = "x";
+                    if ((check[a] == null) || (check[a] == " ")) {
+                        checkOrNot = (" ");
+                    }
+                    System.out.println((a) + ". " + "[" + checkOrNot + "] " + items[a]);
+                }
+            } else if (line.startsWith("mark")) {
+                String strIndex = line.substring(5);
+                int index = Integer.parseInt(strIndex);
```

Comment:
Perhaps having index and strIndex as similar names may be confusing to some people, so can consider using other variable names